src/Services/src/DataPersistence/IDataPersistenceService.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Tanzu.Toolkit.Services.DataPersistence
+{
+    public interface IDataPersistenceService
+    {
+        string ReadStringData(string key);
+        bool WriteStringData(string key, string value);
```

Comment:
What's the purpose of the `bool` return?

Reply:
Idea was that it would indicate success - not currently using it like that but leaving it for now pending a later conversation on standardizing our approach to various kinds of successes/failures